.github/workflows/docker.yml

```diff
@@ -46,13 +48,15 @@ on:
            "fedora-37",
            "fedora-38",
            "fedora-39",
+           "fedora-40",
```

Comment:
Please remove fedora-40 until it is green.

Reply:
Platforms don't magically become green. Developers who work on the portability look at such test runs to find out what needs to be fixed.